HousingSearchListener/V1/Gateway/ApiGateway.cs

```diff
@@ -84,7 +85,7 @@ public void Initialise(string apiName, string configKeyApiUrl, string configKeyA
 
             var responseBody = await response.Content.ReadAsStringAsync().ConfigureAwait(false);
             if (response.IsSuccessStatusCode)
-                return JsonSerializer.Deserialize<T>(responseBody, _jsonOptions);
```

Comment:
Why are we changing it to `DeserializeObject`, this is a common class that is been used for all events